ldes-server-domain/src/test/resources/eventstream/streams/ldes-and-dataset-with-named-views.ttl

```diff
@@ -0,0 +1,70 @@
+@prefix tree: <https://w3id.org/tree#> .
+@prefix ldes: <https://w3id.org/ldes#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix example: <http://example.org/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix custom: <http://example.org/> .
+@prefix mobiliteit: <https://data.vlaanderen.be/ns/mobiliteit#> .
+@prefix col: <http://localhost:8080/collectionName1/> .
+@prefix server: <http://localhost:8080/> .
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+@prefix dct:   <http://purl.org/dc/terms/> .
+@prefix dcat:  <http://www.w3.org/ns/dcat#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix org:   <http://www.w3.org/ns/org#> .
+@prefix legal: <http://www.w3.org/ns/legal#> .
+@prefix m8g:   <http://data.europa.eu/m8g/> .
+@prefix locn:  <http://www.w3.org/ns/locn#> .
+@prefix view1:           <http://localhost:8080/collectionName1/view1/> .
+@prefix view2:           <http://localhost:8080/collectionName1/view2/> .
+
+server:collectionName1 a ldes:EventStream, dcat:Dataset ;
```

Comment:
So i've received the feedback from @WLefever-Cegeka that our application couldn't handle configuring collections and views in one request. I suppose this proves the opposite?

Reply:
This is possible, you can create them at the same time and fetching the config of an eventstream also returns the view configs